src/metabase/util.cljc

```diff
@@ -944,3 +944,23 @@
    (fn [acc x] (if (pred x) (reduced x) acc))
    nil
    coll))
+
+#?(:clj
+   (defmacro case-enum
+     "Like `case`, but explicitly dispatch on Java enum ordinals."
+     [value & clauses]
+     #_ {:clj-kondo/ignore [:discouraged-var]}
+     (let [types (map (comp type eval first) (partition 2 clauses))]
```

Comment:
Isn't there another way to look up the type from a symbol? 😢

Reply:
I found this 🤢 

```clojure
(let [sym 'JsonToken/VALUE_STRING]
  (clojure.lang.Reflector/invokeStaticMethod (resolve (symbol (namespace sym))) 
                                             "valueOf" 
                                             (to-array [(name sym)])))
```

Still better than eval I think, though.

Reply:
`(defn sym->enum [s] 🤮 )`

Reply:
as they say, be wrong on the internet and learn 😁